src/main/java/com/sequenceiq/cloudbreak/converter/RecipeConverter.java

```diff
@@ -8,6 +8,8 @@
 @Component
 public class RecipeConverter extends AbstractConverter<RecipeJson, Recipe> {
 
+    public static final int DEFAULT_TIMEOUT = 5;
```

Comment:
If I recall correctly we were talking about 10 minutes as the default timeout, why was it changed? I think it makes sense that the default timeout is a bit longer.